admin_guide/service_accounts.adoc

```diff
@@ -64,8 +64,29 @@ To allow all service accounts in the *managers* project to edit resources in the
 ----
 $ oc policy add-role-to-group edit system:serviceaccounts:managers -n top-secret
 ----
-// end::sa-user-names-and-groups[]
 
+[[dedicated-admin-granting-service-accounts-access]]
+ifdef::openshift-dedicated[]
+== Granting Service Accounts Access to Dedicated-Admin Roles
+
+As an {product-title} administrator, you can use service accounts to perform any
+actions that require {product-title} *admin* roles.
+
+The {product-title} *admin* service creates the
+xref:../admin_guide/index.adoc#admin-guide-index-dedicated-admin-role[*dedicated-admins
+group*]. This group is granted the roles at the cluster or individual project
+level. Users can be assigned to this group and group membership defines who has
+{product-title} administrator access. However, by design, service accounts
+cannot be added to regular groups.
+
+Instead, the {product-title} *admin* service creates a special project for this
```

Comment:
`{product-title} *admin*` was written by me when I incorrectly assumed that that `{product-title}` translates to "dedicated'. However, since it does not, and this string is expected to be the name of a service, we need to change it to `dedicated-admin`